tests/unit/plugins/modules/ciscosmb/test_ciscosmb_facts-SG350-28-K9.py

```diff
@@ -99,11 +99,10 @@ def test_ciscosmb_facts_default(self):
             result['ansible_facts']['ansible_net_serialnum'], 'ABC1234567A'
         )
 
-        with self.assertRaises(KeyError) as cm:
-            len(result['ansible_facts']['ansible_net_stacked_models'])
-        the_exception = cm.exception
-        print(the_exception)
-        self.assertIsInstance(the_exception, KeyError)
+# TODO         with self.assertRaises(KeyError) as cm:
```

Comment:
Do you want to raise an issue to track this?

Reply:
No, I will work no unit tests later.

Currently there are full units only for one type of devices SG550.